code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -982,20 +989,61 @@
 		if(!embeddies.isEmbedHarmless())
 			bleed_rate += 0.25
 
-	for(var/datum/wound/wound as anything in wounds)
-		bleed_rate += wound.blood_flow
-
-	if(owner.body_position == LYING_DOWN)
-		bleed_rate *= 0.75
+	for(var/datum/wound/iter_wound as anything in wounds)
+		bleed_rate += iter_wound.blood_flow
 
-	if(grasped_by)
-		bleed_rate *= 0.7
+	if(!ignore_modifiers)
+		if(owner.body_position == LYING_DOWN)
+			bleed_rate *= 0.75
+		if(grasped_by)
+			bleed_rate *= 0.7
 
 	if(!bleed_rate)
 		QDEL_NULL(grasped_by)
 
 	return bleed_rate
 
+// how much blood the limb needs to be losing per tick (not counting laying down/self grasping modifiers) to get the different bleed icons
+#define BLEED_OVERLAY_LOW 0.5
+#define BLEED_OVERLAY_MED 1.5
+#define BLEED_OVERLAY_GUSH 3.25
+
+/obj/item/bodypart/proc/update_part_wound_overlay()
```

Comment:
Can you make the other paths explicitly return FALSE? Unexpected FALSE over null causes weird gotchas, and is just a weird signature.